gigaword-indexer/src/main/java/edu/isi/vista/gigawordIndexer/IndexGigawordWithElasticSearch.java

```diff
@@ -125,6 +127,12 @@ private static RestHighLevelClient buildElasticSearchClient(Parameters parameter
   private static void index(RestHighLevelClient client, Path file, String indexName) throws IOException {
     log.info("Indexing {}", file.toAbsolutePath());
 
+    // If file ends with .xml.gz, this may be the wrong version of Gigaword.
+    if (file.toString().toLowerCase().endsWith(".xml.gz")) {
```

Comment:
This works, but `file.getName().toLowerCase()....` would be cleaner.

Reply:
Both `getName(index)` and `getFileName()` return a `Path`.

Reply:
Oh, it's a `Path` object, not a `File`.  `toString` is fine then